chia/protocols/harvester_protocol.py

```diff
@@ -71,6 +102,8 @@ class RespondSignatures(Streamable):
     local_pk: G1Element
     farmer_pk: G1Element
     message_signatures: List[Tuple[bytes32, G2Element]]
+    include_source_signature_data: bool = False
+    farmer_reward_address_override: Optional[bytes32] = None
```

Comment:
I think you should update `tests/util/network_protocol_data.py` to make these fields not be `None` in the tests. That would increase the coverage of the protocol stability unit tests.

Not just these fields, but all new fields you're adding to protocol messages that are `Optional` and defaults to `None`